modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -73,6 +73,12 @@ class Test_ONNX_layers : public DNNTestLayer
     }
 };
 
+TEST_P(Test_ONNX_layers, InstanceNorm)
+{
+
+    testONNXModels("instancenorm", npy, 1, 1, false, false);
```

Comment:
Please do not forget to open a PR with test data at https://github.com/opencv/opencv_extra